script/index.js

```diff
@@ -252,4 +295,31 @@ cardsArray.forEach((list) => {
     document.querySelector('.desktpopupcontainer').classList.toggle('expanded');
     document.querySelector('.popupcontainer').classList.toggle('extended');
   });
+<<<<<<< HEAD
+
+})
+
+// form validation
+
+function valid() {
+  const email = document.getElementById('email').value;
+  const validatedEmail = email.toLowerCase();
+  if (validatedEmail === email) {
+    document.querySelector('.error').textContent = '';
+    return true;
+  }
+  document.querySelector('.error').textContent = 'Email address should always be in Lowercase.';
+  return false;
+}
+
+// stop button default action
+const form = document.querySelector('form');
+form.addEventListener('submit', (event) => {
+  if (valid() === false) {
+    event.preventDefault();
+  }
+});
```

Comment:
- [ ] As in the project requirement, if a user submits the contact form with the email field containing uppercase characters an error message should show up but that is not the case for you. When I submit a contact form with the email field containing uppercase characters it submits correctly, please kindly fix it 🙃 🙃

Reply:
ok, i will do so now